CONTRIBUTING.rst

```diff
@@ -1,6 +1,11 @@
 Contribution guidelines
 =======================
 
+This guide is describing rules how to get your contributions to Anaconda. However, if you are seeking
+for help how to implement changes to Anaconda please follow our
```

Comment:
for help with implementing changes in Anaconda, please follow our
(note I'm not native speaker, so maybe that's still funny)